datasource-local/src/main/java/cmm/apps/esmorga/datasource_local/database/Converter.kt

```diff
@@ -20,7 +20,7 @@ class ZonedDateTimeConverter {
 
     @TypeConverter
     fun toTimestamp(date: ZonedDateTime): String {
-        return date.format(DateTimeFormatter.ofPattern(DATE_FORMAT).withZone(TimeZone.getDefault().toZoneId()))
+        return date.format(DateTimeFormatter.ofPattern(DATE_FORMAT))
```

Comment:
Do we need the fromTimestamp TypeConverter now?

Reply:
I also removed toTiemeStamp() method that it was not used anywhere now